### DISCUSSION TITLE
Facing an issue in one of our console applications after quartz is upgraded from version 2.2 to 3.5Error says-Unexpected Error Could not load file or assembly 'System.ValueTuple, Version=4.0.1.0,Culture=neutral,PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies

### DISCUSSION THREAD

**Discussion by Rahul2884** [quartznet/quartznet#1894] [Q&A]
Facing an issue in one of our console applications after quartz is upgraded from version 2.2 to 3.5..Error says- "Unexpected Error Could not load file or assembly 'System.ValueTuple, Version=4.0.1.0,Culture=neutral,PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependency"
How can this be fixed?
@lahma

**Comment by lahma** [comment]
Generally sounds like a problem running against old dull framework, like pre net472 era. Google has probably a lot of hits as this is not Quartz specific.

**Comment by baimengyuan** [comment]
using System.ValueTuple   use System.ValueTuple.dll
